sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -540,8 +540,14 @@ object QueryCompilationErrors {
       s"rename temporary view from '$oldName' to '$newName': destination view already exists")
   }
 
-  def databaseNotEmptyError(db: String, details: String): Throwable = {
-    new AnalysisException(s"Database $db is not empty. One or more $details exist.")
+  def cannotDropNonemptyDatabaseError(db: String, details: String): Throwable = {
```

Comment:
Since we can remove the `details` parameter now.

Reply:
Sorry, I missed the call `cannotDropNonemptyDatabaseError(db, "tables")` and `cannotDropNonemptyDatabaseError(db, "functions")` in `InMemoryCatalog` file. So I think keep origin error message and also mention `use CASCADE`.

Reply:
Actually, I think `One or more $details exist` is a repeat of `non-empty database`, and it's OK to remove it, to keep consistent with v2.

Reply:
Updated. Thank you @cloud-fan!